test/Makefile

```diff
@@ -2,6 +2,11 @@ TARGETS=test all compile testll
 
 TDIRS= $(sort $(basename $(notdir $(wildcard ../compiler/testSuite/*))))
 
+COMPILER_JAR=../compiler/target/bin/nballerina.jar
```

Comment:
Remove defns of COMPILER_JAR and JAVA. make in this directory assumes COMPILER_JAR exists.